common/display/displayplanemanager.cpp

```diff
@@ -111,6 +111,9 @@ std::tuple<bool, DisplayPlaneStateList> DisplayPlaneManager::ValidateLayers(
   auto layer_begin = layers.begin();
   auto layer_end = layers.end();
   bool render_layers = false;
+  // Below flag takes care of condition,when the cursor layer is assigned the
+  // lower zorder
+  bool cursor_plane_enable = true;
```

Comment:
Hi Kalyan,
In this particular UC, we see 2 layers including cursor layer.
I have checked the Z-order of the layers in the presentDisplay call. cursor layer is coming at the lower z-order than the other layer.
I have missed the case when we get only cursor layer, that case I will assign to cursor layer.
Thanks